data/json/items/comestibles/raw_fruit.json

```diff
@@ -197,6 +197,27 @@
     "smoking_result": "dry_fruit",
     "vitamins": [ [ "vitA", 1 ], [ "vitC", 5 ], [ "calcium", 2 ], [ "iron", 3 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "wintergreen_berry",
+    "name": { "str": "wintergreen berry", "str_pl": "wintergreen berries" },
+    "weight": "3 g",
+    "color": "red",
+    "spoils_in": "5 days",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "healthy": 1,
+    "calories": 20,
```

Comment:
Is no quench intentional?

Reply:
Yes. The problem is that I never tasted wintergreen berries in real life and as far as I can tell those berries are unlike blueberries, especially towards the end of winter where they apparently tend to dry out.